src/components/LongTextMarkdownDisplay.tsx

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+
+import { 
+  Flex,
+  Center,
+  Text,
+  Image,
+  Button,
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalHeader,
+  ModalCloseButton,
+  ModalBody,
+  ModalFooter,
+  useDisclosure
+} from "@chakra-ui/react";
+
+import ReactMarkdown from 'react-markdown'
+import remarkGfm from 'remark-gfm'
+      
+const LongTextMarkdownDisplay = ({
+  longTextMarkdown,
+  header
+}: {
+  longTextMarkdown: string;
+  header: string;
+}) => {
+
+  const { isOpen, onOpen, onClose } = useDisclosure();
+
+  return (
+    <Flex flexDir="column">
+      <Text fontSize="m" maxH="120px" overflow="hidden">{longTextMarkdown}</Text>
+      <Center>
+        <Text fontSize="sm" color="#8ab4f8" onClick={onOpen}>read more...</Text>
```

Comment:
If text fits to 120px then "read more..." is not needed, right? 
Ishkhan suggested for the voting page this approach: if text not bigger than 6 lines - display it, if not - display 4 lines and add 'read more'. For the voting page I decided that it would be 500 and 400 characters respectively.  
This is my component:
https://github.com/great-wyrm/frontend/blob/main/src/components/TextWithPopup.tsx